### DISCUSSION TITLE
Not able to setup project locally.

### DISCUSSION THREAD

**Discussion by Rohit-Sharmaa** [OWASP/threat-dragon#794] [General] [answered]
I am new to open source and i am facing issue how to setup this project can someone help me with this ??
Please

**Comment by jgadsden** [comment, marked as answer]
Hello @Rohit-Sharmaa - a good starting point is the install instructions in the [Threat Dragon documentation](https://owasp.org/www-project-threat-dragon/docs-2/install-desktop/)
